packages/flutter_tools/lib/src/android/gradle_utils.dart

```diff
@@ -75,6 +75,11 @@ RegExp(
     multiLine: true
 );
 
+// This matches uncommented minSdkVersion lines in the module-level build.gradle
+// file which have minSdkVersion 16,17, or 18 (the Jelly Bean api levels).
+final RegExp jellyBeanMinSdkVersionMatch =
+RegExp(r'(?<=^\s*)minSdkVersion 1[678](?=\s*(?://|$))', multiLine: true);
```

Comment:
nit:
```suggestion
final RegExp jellyBeanMinSdkVersionMatch =
  RegExp(r'(?<=^\s*)minSdkVersion 1[678](?=\s*(?://|$))', multiLine: true);
```

Reply:
Changed, and also adjusted the formatting of the other regexps defined in gradle_utils to match this format